### PR TITLE
Update WA CKAN

### PR DESCRIPTION
**Pull request by steve9164**
Fixes https://github.com/TerriaJS/nationalmap/issues/946 and fixes https://github.com/TerriaJS/nationalmap/issues/940.

## Note
For the following two reasons, datasets of `GeoJSON`, `WFS`  and `Esri Feature Service` are excluded from WA CKAN catalogue.

  1. Most of `GeoJSON` datasets  have property of `access_level:public_login`. The `national map` will not be able to visualise them for unauthenticated users. 
  2. For all `WFS` datasets, the `national map` reports error `TypeError: Cannot read property '0' of undefined`.

### CODE REVIEW

**Comment by steve9164** [comment]
Test at https://nationalmap.gov.au/#clean&https://raw.githubusercontent.com/TerriaJS/NationalMap-Catalog/master/build/nm.json

File: datasources/includes/WA.ejs [hunk 1]
```diff
@@ -1,7 +1,17 @@
     {
       "type": "ckan",
       "name": "Western Australian Government",
-      "url": "http://catalogue.beta.data.wa.gov.au/",
+      "url": "https://catalogue.data.wa.gov.au/",
       "groupBy": "organization",
-      "filterQuery": [ "q=data_homepage:*_Public_Services*&fq=res_format:WMS" ]
+      "filterQuery": [ 'q=access_level:open&fq=res_format:(GeoJSON OR "OGC WMS" OR "OGC WFS" OR WFS OR "API ArcGIS Server Feature Service" OR "API ArcGIS Server Map Service" OR "API ArcGIS Map Service" OR "Esri REST")' ],
```
Comment (mwu2018) [inline datasources/includes/WA.ejs]: The query will return results with `access_level:open` at top level. However, there are mixed `access_level:open` and `access_level:open_login` in the `resources` list. The latter only grants access to authenticated users which is not supported by the `National Map`. 
Besides, resource list includes many formats like CSV, SHP, PDF, etc.

**Comment by mwu2018** [comment]
With https://nationalmap.gov.au/#clean&https://raw.githubusercontent.com/TerriaJS/NationalMap-Catalog/wa-ckan/build/nm.json, this resource has `access_level:open`
Capture1

but got error message:

Capture2

File: build/nm_big.json [hunk 2]
```diff
@@ -19802,14 +19802,13 @@
...
       ],
       "includeCsv": true,
-      "includeGeoJson": true,
+      "includeGeoJson": false,
       "includeKml": true,
       "includeWms": true,
-      "includeWfs": true,
+      "includeWfs": false,
       "includeEsriMapServer": true,
       "includeEsriFeatureServer": true,
```
Comment (steve9164) [inline build/nm_big.json]: You should exclude ESRI Feature Server too.

Comment (mwu2018) [reply build/nm_big.json]: done.

**Pull request merged**